src/main/java/com/dev/theater/dao/MovieDao.java

```diff
@@ -0,0 +1,12 @@
+package com.dev.theater.dao;
+
+import com.dev.theater.library.Dao;
+import com.dev.theater.model.Movie;
+import java.util.List;
+
+@Dao
```

Comment:
should we use Dao annotation above interface?

Reply:
deleted